tests/store/tracking/test_file_store.py

```diff
@@ -315,13 +316,22 @@ def test_search_experiments_pagination(store):
     assert experiments.token is None
 
 
-class TestFileStore(unittest.TestCase, AbstractStoreTest):
+class TestFileStoreHelper:
```

Comment:
Let's remove this class.

Reply:
Added this class as there were a lot of data variables being created in the `_create_root()` function. Can you suggest a better way to manage these variables without creating a class?

Reply:
If so, let's remove `_create_root` and refactor each test.

Reply:
Yes, we can do that but the only drawback I see is that there will be a lot of redundant operations involved in recreating runs/experiments for every test.

Reply:
I think that's fine. Creating required resources in a test is easier to understand and veriry than creating required resources in a fixture.

```
def test_search_runs(fixture):
    runs = search_runs()  # where do we create runs...?
    assert len(runs) == 3
```

```
def test_search_runs(fixture):
    create_run("a")
    create_run("b")
    runs = search_runs()
    ...
```

Reply:
Understood! I'll make the required changes.